soundcard/pulseaudio.py

```diff
@@ -614,6 +621,8 @@ def __exit__(self_, exc_type, exc_value, traceback):
     _pa_stream_peek = _lock(_pa.pa_stream_peek)
     _pa_stream_drop = _lock(_pa.pa_stream_drop)
     _pa_stream_connect_playback = _lock(_pa.pa_stream_connect_playback)
+    _pa_stream_update_timing_info = _lock(_pa.pa_stream_update_timing_info)
+    _pa_stream_get_latency = _lock(_pa.pa_stream_get_latency)
```

Comment:
I think these two declarations should go into the `_Stream` class.

Reply:
I agree with the idea as it will only be called from the `_Stream` class but in the current code organization, we cannot call `_lock` and `_lock_and_block` from the `_Stream` class as it uses `self._lock_mainloop()`, with `self` being only `_PulseAudio`. (Same for `_block_operation` and `_pa_operation_unref`). 
Am I missing something?

Reply:
You are right. These functions are in the right place. The pull request made it seem like they were part of the `_Player`. I'm sorry about the confusion.